src/server/scripts/Spells/spell_warlock.cpp

```diff
@@ -773,9 +772,9 @@ class spell_warl_life_tap : public SpellScript
         Player* caster = GetCaster()->ToPlayer();
         if (Unit* target = GetHitUnit())
         {
-            int32 damage = GetEffectValue() + LIFE_TAP_COEFFICIENT;
-            int32 damage2Mana = GetEffectValue();
-            int32 mana = int32(damage2Mana + (caster->GetInt32Value(PLAYER_FIELD_MOD_DAMAGE_DONE_POS + static_cast<uint8>(SPELL_SCHOOL_SHADOW)) * 0.5f));
+            int32 spellEffect = GetEffectValue();
+            int32 damage = spellEffect;
```

Comment:
```suggestion
            int32 damage = GetEffectValue();
```

Reply:
Had it like that for just one call to GetEffectValue()
spelleffect also appears in the managain calculation